### DISCUSSION TITLE
Use Symbol in v-if

### DISCUSSION THREAD

**Discussion by sqllyw** [vuejs/petite-vue#138] [Q&A] [answered]
Hi,
can we use Symbol in v-if?
```
  const IDLE = Symbol()
```
 somewhere in template
```
 <div v-if="state==IDLE"></div>
```
got IDLE not defined, a simple non working code:
https://jsfiddle.net/4hpswLav/1/

**Comment by halivert** [comment, marked as answer]
You should pass the `IDLE` symbol in the `createApp` object.

**Reply by sqllyw** [reply]
good solution, here is an updated jsfiddle:
https://jsfiddle.net/9qxswdam/
thanks
